scriptmodules/supplementary/webgui.sh

```diff
@@ -0,0 +1,38 @@
+
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="webgui"
+rp_module_desc="RetroPie WebGUI on port 8080"
```

Comment:
As we already have some other "guis" it might be good to include your name here and some more information in a rp_module_help.